tests/lib.rs

```diff
@@ -732,3 +732,40 @@ fn retain() {
     vs.sort();
     assert_eq!(v, &*vs);
 }
+
+#[test]
+fn get_one_short() {
+    let x = ('x', 42);
+
+    let (r, mut w) = evmap::new();
+
+    w.insert(x.0, x);
+
+    assert_match!(r.get_one(&x.0), None);
+
+    w.refresh();
+
+    assert_match!(r.get_one(&x.0).unwrap().as_ref(), ('x', 42));
+}
+
+#[test]
+fn get_one_long() {
+    let x = ('x', 42);
+
+    let (r, mut w) = evmap::new();
+
+    // Add 32 items to meet the BAG_THRESHOLD and
```

Comment:
Gross. I hate relying on this implementation detail here. Wouldn't want this test to lose its flair if `BAG_THRESHOLD` changes. And the worst part is it'll just silently keep passing. So I'm willing to bet this test just isn't worth having in here?

Reply:
I think what I would do here is make `BAG_THRESHOLD` be `pub`, but `#[doc(hidden)]`, and then add to its doc comment that it is only `pub` to be used in tests.

Reply:
Great! I was afraid of exposing that but `doc(hidden)` seems like a good way to expose it without encouraging its use 👍 Two quick questions on this:

1) I've never used `rustdoc` before. To add the attribute _and_ the rustdoc is it like:
```rs
/// Comment here
#[doc(hidden)]
pub ...
```
? Or are the comments and attribute switched? Or is it some other configuration? In all the examples I see it's like:
```rs
// Non-doc-comment here
#[doc(hidden)]
pub ...
```

2) I can make `BAG_THRESHOLD` `pub` but to access it in `tests/lib.rs` I think I have to also make the [`values` module public](https://github.com/jonhoo/rust-evmap/blob/master/src/lib.rs#L248) or add it to the [`pub use` for `values`](https://github.com/jonhoo/rust-evmap/blob/master/src/lib.rs#L249). I'm assuming of those two, the second is preferable but I'm just checking first in case neither are okay!

Reply:
Ah, for 1, I think the order actually doesn't matter. Though my personal preference is doc comment, then attributes.

For 2, I'd just move the definition of `BAG_THRESHOLD` to the root of the crate!

Reply:
Awesome, thanks for all that help. Changes in 40062d0c28bb844163fdb6e63e3dbfa602a023d0. I was a little uncertain if I should write better comments for `BAG_THRESHOLD` since it's defined farther from where it's used but felt silly writing more doc comments that were hidden. I kind of wanted to write non-doc comments 🤔 Which kind of matches the other examples I see so now I understand that. But I went with what we'd already mentioned and am quite willing to update it any which way!

Reply:
I think adding a bit more of a generous doc comment would be good. Something like:

```rust
/// This value determines when a value-set is promoted from a list to a HashBag.
```

Reply:
I ended up making this private again since I moved the tests to `src/values`. I hope that's okay. I left in the doc comment 🤷. e9bd66964fcca1221bbb6b3c42e22e8f8246b062